astroquery/irsa/core.py

```diff
@@ -451,10 +451,10 @@ def _parse_coordinates(coordinates):
             warnings.warn("Coordinate string is being interpreted as an "
                           "ICRS coordinate.")
         except u.UnitsError as ex:
-            warnings.warn("Only ICRS coordinates can be entered as strings\n"
+            warnings.warn("Only ICRS coordinates can be entered as strings \n"
                           "For other systems please use the appropriate "
                           "astropy.coordinates object")
-            raise ex
+
```

Comment:
does this Exception still need to be raised?  i.e., can the code proceed from this point?

Reply:
can you elaborate a little pls...

Reply:
Can the code beyond this point execute given the failure here?  An exception is being raised, therefore the code beyond this point has never been executed.  Make sure that it can.  Most likely, the `raise ex` is here intentionally because the code beyond this line _cannot_ be run once you've reached this line.